SDKs/python_sdk/import_products_from_shopify_csv/import_products.py

```diff
@@ -0,0 +1,131 @@
+                cprint("Grams: %s" % grams, "cyan")
+                weight_unit = shopify_product.get('Variant Weight Unit')
+                cprint("Weight Unit: %s" % weight_unit, "cyan")
+
+                properties = {
+                    'vendor': vendor,
+                    'grams': grams,
+                    'weight_unit': weight_unit
+                }
+
+                cprint("=========================== Tags ===========================")
+                tags = [tag for tag in shopify_product.get('Tags').split(',')]
+                tags.append(shopify_product.get('Type'))
+                cprint("Tags: %s" % tags, "cyan")
+
+                cprint("=========================== Other ===========================")
+                active = parse_boolean_param(shopify_product.get('Published'))
+                cprint("Active: %s" % active, "cyan")
+                name = shopify_product.get('Title')
+                cprint("Name: %s" % name, "cyan")
+                description = shopify_product.get('Body (HTML)')
+                cprint("Description: %s" % description, "cyan")
+                price = float(shopify_product.get('Variant Price'))
+                cprint("Price: %s" % price, "cyan")
+                external_reference = shopify_product.get('Handle')
+                cprint("Handle | External Reference: %s" % external_reference, "cyan")
+                image_url = shopify_product.get('Image Src')
+                cprint("Image Url: %s" % image_url, "cyan")
+
+                body = {
+                    'active': active,
+                    'name': name,
+                    'description': description,
+                    'price': price,
+                    'external_reference': external_reference,
+                    'tags': tags,
+                    'properties': properties,
+                    'image_url': image_url
+                }
+
+                builton.product().create(body=body)
+                cprint("================== SUCCESSFULLY IMPORTED ===================", "green")
+
+            except Exception as error:
+                cprint("========================== ERROR ===========================", "red")
+                cprint("something wrong happened: %s" % error, "red")
+                cprint("product handle: %s" % shopify_product.get('Handle'), "yellow")
+                cprint("product csv: %s" % shopify_product, "yellow")
+                cprint("============================================================")
+
+                continue
+        cprint("============================================================")
+        cprint("======================= IMPORT DONE ========================")
+        cprint("============================================================")
+
+
+def parse_boolean_param(param):
+    if param is None:
+        return param
+    if (isinstance(param, int) and param == 1) or (param.strip().lower() in ['1', 'true']):
```

Comment:
if `param != 0`, it will try to `strip()` it and it will raise an `AttributeError`

Reply:
Well, it should not be > 1... I mean `shopify_product.get('Published')` should return `0` or `1` because it's a Boolean logic

Reply:
So, if it returns 0, it would fail